final_task/rss_reader/rss_item.py

```diff
@@ -0,0 +1,36 @@
+import html
+import json
+from dataclasses import dataclass
+
+
+@dataclass
+class RssItem:
+    '''
+    Represents a single news from RSS channel
+
+    source: stores link to rss channel of the news
+    date: stores date on YYYY%MM%DD format
+    '''
+    title: str
+    published: str
+    link: str
+    media: str
+    source: str
+    date: str
+
+    @classmethod
+    def from_dict(cls, item_dict) -> 'RssItem':
+        return cls(
+            title=item_dict['title'],
+            published=item_dict['published'],
+            link=item_dict['link'],
+            media=item_dict['media'],
+            source=item_dict['source'],
+            date=item_dict['date']
+            )
```

Comment:
Check out function `asdict` from `dataclass` lib. It is better to try to use it here instead

Reply:
If I got it correct, `asdict` converts a class to dict, while `from_dict` creates the class from dict. I'm not sure that I could use `asdict` here

Reply:
Yes, my bad. I misunderstood what method does. Still you can use unpacking here, instead of passing each argument by name

Reply:
Will unpacking be suitable if I'm going to process some fields in this method?

Reply:
Well, it depends. You can do something like this:
```
item_dict['field'] = do_some_stuff(item_dict['field'])
.....
return cls(**item_dict)
```
But if you are going to change many fields in the dict, then it is better to leave it as is. See for yourself and use what do you think is more readable

Reply:
Ok, thanks